src/AlgebraicGeometry/Schemes/AffineAlgebraicSet/Objects/Constructors.jl

```diff
@@ -3,28 +3,25 @@
 ########################################################
 
 @doc raw"""
-    affine_algebraic_set(X::Spec; check::Bool=true) -> AffineAlgebraicSet
+    affine_algebraic_set(X::Spec; is_reduced=false, check=true) -> AffineAlgebraicSet
 
-Convert `X` to an `AffineAlgebraicSet` by taking the underlying reduced scheme.
+Convert the `X` to an `AffineAlgebraicSet` by considering its reduced structure.
```

Comment:
The inserted "the" seems odd to me. Perhaps you meant "Convert the NOUN `X` to" with a missing NOUN to be inserted? Otherwise, remove "the" again?